webhook_server/libs/handlers/push_handler.py

```
@@ -69,11 +70,98 @@ async def process_push_webhook_data(self) -> None:
         else:
+            # Non-tag push - check if this is a push to a branch that could be a base for PRs
             self.logger.step(  # type: ignore[attr-defined]
                 f"{self.log_prefix} {format_task_fields('push_processing', 'webhook_event', 'processing')} "
-                f"Non-tag push detected, skipping processing",
+                f"Processing branch push event",
             )
 
+            # Check if retrigger is enabled (not None or empty list)
+            retrigger_config = self.github_webhook.retrigger_checks_on_base_push
+            if not retrigger_config:
+                self.logger.debug(f"{self.log_prefix} retrigger-checks-on-base-push not configured, skipping")
```

Ruff (0.14.10)

Logging statement uses f-string

(G004)